website/client/components/chat/copyAsTodoModal.vue

```diff
@@ -45,7 +45,7 @@ export default {
         type: 'todo',
         notes,
       };
-      this.task = taskDefaults(newTask);
+      this.task = taskDefaults(newTask, message.user);
```

Comment:
I think message.user here is a string not an object, you can access it from the store `this.$store.state.user.data`